bpf/lib/lb.h

```diff
@@ -1386,9 +1479,9 @@ lb4_select_backend_id(struct __ctx_buff *ctx,
 #elif LB_SELECTION == LB_SELECTION_MAGLEV
 static __always_inline __u32
 lb4_select_backend_id(struct __ctx_buff *ctx __maybe_unused,
-		      struct lb4_key *key __maybe_unused,
-		      const struct ipv4_ct_tuple *tuple,
-		      const struct lb4_service *svc)
+					  struct lb4_key *key __maybe_unused,
+					  const struct ipv4_ct_tuple *tuple,
+					  const struct lb4_service *svc)
```

Comment:
nit: formatting is wrong (!= kernel style), this hunk could be removed and code left as-is